cli/utilities/file.py

```diff
@@ -0,0 +1,17 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def copy_file(file_name, src, dest):
```

Comment:
For this functionality Ceph method is already avaialble - https://github.com/red-hat-storage/cephci/blob/master/ceph/ceph.py#L2091

Reply:
Done. Removed this file.